synapse/models/files.py

```diff
@@ -235,13 +272,103 @@ def getModelDefs(self):
                         'doc': 'The size of the executable file according to the PE file header.'}),
 
                     ('mime:pe:imphash', ('guid', {}), {
-                        'doc': 'The PE import hash of the file as calculated by Vivisect; this method excludes '
+                        'doc': 'The PE import hash of the file as calculated by pefile; '
+                               'https://github.com/erocarrera/pefile'}),
+
+                    ('mime:pe:imphash:vtx', ('guid', {}), {
```

Comment:
`viv` not `vtx`

Reply:
This feels odd from a modeling perspective since we're encoding two ways to compute the save data at two different modeling hierarchies